### ISSUE TITLE
Loss functions explanation

### ISSUE THREAD

**Issue by juanjucm** [runqitian/CR-CNN#1]
Hi @runqitian :) 

Firs of all, thanks for this awesome implementation of the paper. I have a question regarding loss functions that you have implemented:
In the paper, the authors define in Equation 1 the Loss Function used to train the network. I've seen that you encoded this function in "class_loss_func". However, what does "other_loss_func" do? Since I'm not able to find anything like that it in the original paper.

Thanks so much!
